Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM resin/nuc-node:6-20161215
+FROM balenalib/intel-nuc-node:6-stretch-build-20190127
 
 # Install PIP, Robot Framework, Resin-cli and Etcher-cli
-RUN apt-get update && apt-get install -y qemu-system-x86 rsync qemu-kvm minicom libftdi-dev python-pip && \
+RUN apt-get update && apt-get install -y qemu-system-x86 rsync qemu-kvm minicom libftdi-dev python-pip python-setuptools python-wheel systemd && \
```

Comment:
you add systemd, how is it used? The `balenalib` base image doesn't start it up normally, as much as I know.

Reply:
@imrehg The Dockerfile uses systemctl. The systemd package provides it.

Reply:
Makes sense. Have you tried that things continue to work inside the container after this change? (I assume yes, but wanted to ask :)

Reply:
@imrehg yea did run a test run. seemed happy with qemux86-64. But not with qemux86..